src/SFA.DAS.AANHub.Application.UnitTests/Apprentices/Commands/MemberTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.Apprentices.Commands;
+using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.Apprentices.Commands
+{
+    [TestFixture]
+    public class MemberTest
+    {
+        [Test, AutoMoqData]
+        public void Apprentice_PopulatesMemberFromCreateApprenticeMemberCommand(CreateApprenticeMemberCommand createApprenticeMemberCommand)
+        {
+            var command = (Member)createApprenticeMemberCommand;
+
+            Assert.That(command, Is.Not.Null);
+
+            command.Should().BeEquivalentTo(createApprenticeMemberCommand, c => c
```

Comment:
you don't need both BeEquivalent and Assert.AreEqual statements, they are doing the same job here.  Take one out.

Reply:
Removed `Assert.AreEqual`